app/NoteAndAccidentalInput/page.tsx

```diff
@@ -228,19 +139,19 @@ const CreateAndEraseNotesFromStave = () => {
       <div ref={container} onClick={handleClick} />
       <CheckNumBeatsInMeasure
         tooManyBeatsInMeasure={state.tooManyBeatsInMeasure}
-        openEnterNotes={enterNotes}
+        openEnterNotes={dispatch}
       />
       <CheckIfNoteFound
         noNoteFound={state.noNoteFound}
-        openEnterNotes={enterNotes}
+        openEnterNotes={dispatch}
       />
       <div className="mt-2 ml-3">
```

Comment:
Just a thought: Maybe we should start to move away from Tailwind since we're barely using it and we have made Material UI our primary CSS framework. Eventually, it might be good to convert all Tailwind to Material UI then uninstall Tailwind dependencies. Here, for example, you could have it be a Box with the same mt and ml prop settings.

Reply:
agreed